### ISSUE TITLE
Research how to & implement build FreeBSD Target on GitHub Actions

### ISSUE THREAD

**Issue by ghost** [input-leap/input-leap#331]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/331
> 
> Issue created on: 2019-06-02 by @nelsonjchen
> Issue last updated on: 2019-06-02

There are rust CI toolkits that will build and test your Rust program on FreeBSD. Maybe we can look at those and see if theres a way to build barrier on Azure for FreeBSD. 

Or some cross compiling stuff, 🤷‍♂️. 

---

> Commented on: 2019-06-02 by @AdrianKoshka

Oh yeah, I actually totally forgot we have a FreeBSD target, mostly because freeBSD users don't seem to post issues here, haha.

**Comment by Onepamopa** [comment]
Has there been any progress towards a port for freebsd?

**Comment by shymega** [comment]
No, nothing. v3.0.0 isn't out yet. We are hopefully moving to GitHub Actions, and maybe we can later on add FreeBSD targets with this: https://github.com/marketplace/actions/freebsd-vm

If you have any build instructions from experiments, please make a PR here: https://github.com/input-leap/wiki-prs

**Comment by Onepamopa** [comment]
> No, nothing. v3.0.0 isn't out yet. We are hopefully moving to GitHub Actions, and maybe we can later on add FreeBSD targets with this: https://github.com/marketplace/actions/freebsd-vm
> 
> If you have any build instructions from experiments, please make a PR here: https://github.com/input-leap/wiki-prs

I don't have any, but considering Barrier is working ~fine on freebsd, you can use what's in the existing port to try to build input-leap.

**Comment by sithlord48** [comment]
we no longer use azure this should be changed to github actions or closed.

**Comment by shymega** [comment]
Agreed, I'll update the issue title.

**Comment by shymega** [comment]
I've also removed this from the v3.0.0 milestone, so we can get a release out of the door.
